scripts/machine_translation/README.md

```diff
@@ -30,30 +30,38 @@ python3 train_transformer.py \
     --save_dir transformer_base_wmt2014_en_de_${SUBWORD_ALGO} \
     --cfg transformer_base \
     --lr 0.002 \
-    --batch_size 2700 \
-    --num_averages 5 \
-    --warmup_steps 4000 \
+    --sampler BoundedBudgetSampler \
+    --max_num_tokens 2700 \
+    --max_update 15000 \
+    --save_interval_update 500 \
+    --warmup_steps 6000 \
     --warmup_init_lr 0.0 \
     --seed 123 \
     --gpus 0,1,2,3
 ```
 
-Use the average_checkpoint cli to average the last 10 checkpoints
+Or training via horovod
+```
+mpirun -np 4 -H localhost:4 python3 train_transformer.py \
+    --comm_backend horovod \
+    ...
+```
```

Comment:
Do you have any special reason to use mpirun instead of horovodrun? @ZheyuYe

Reply:
At first I cant run the scripts via 'horovodrun' and switched to 'mpirun'，probably because I didn't install it properly. Let me re-try 'horovodrun'.

Reply:
changed mpirun to horovodrun